src/core/tests/type_prop/bitwise_ops.hpp

```diff
@@ -0,0 +1,1020 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+#include "common_test_utils/test_assertions.hpp"
+#include "common_test_utils/type_prop.hpp"
+#include "openvino/core/dimension_tracker.hpp"
+#include "openvino/op/util/attr_types.hpp"
+
+using namespace ov;
+using namespace testing;
+
+template <class T>
+class BitwiseOperator : public testing::Test {};
+
+TYPED_TEST_SUITE_P(BitwiseOperator);
+
+TYPED_TEST_P(BitwiseOperator, default_constructor_integer) {
+    auto A = std::make_shared<ov::op::v0::Parameter>(element::i32,
```

Comment:
```suggestion
    auto a = std::make_shared<ov::op::v0::Parameter>(element::i32,
```
Usually variables have `lower_case` convention. Unify naming for all variables here.

Reply:
Done